integration-tests/src/main/java/io/druid/testing/DockerConfigProvider.java

```diff
@@ -59,6 +59,12 @@ public String getBrokerHost()
       }
 
       @Override
+      public String getHistoricalHost()
+      {
+        return dockerIp + ":8083";
```

Comment:
agreed - that's why a previous PR added ConfigFileConfigProvider, which allows host and port to be configured for each node type, and other general configuration information.